CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [1.18.43]
+### Fixed
+- `<s>` now supported as the first token in a multi-word negative constraint
```

Comment:
[nit] could make this a bullet point and the item in 1.18.42 as well?

Reply:
Sorry, I'm not sure I understand this one. This improvement doesn't apply to 1.18.42, right?

Reply:
I should be more clear: the description is missing a dash "-", so does the previous entry in the change log (length calculation).

Reply:
Where is the dash missing? This is what I see right now:

```
## [1.18.43]
### Added
- `<s>` now supported as the first token in a multi-word negative constraint
  (e.g., '<s> I think' to prevent a sentence from starting with 'I think')
### Fixed
- Bugfix in resetting the state of a multiple-word negative constraint

## [1.18.42]
### Changed
- Simplified gluon blocks for length calculation
```